indico/modules/events/papers/templates/emails/judgment_to_author.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'events/papers/emails/base.html' %}
+
+{% block subject_message %}Paper assessment{% endblock %}
+
+{% block header_subtitle %}Paper assessment{% endblock %}
+
+{% block header_subtitle_ref %}ref #{{ contribution.friendly_id }}{% endblock %}
+
+{% block content_header %}
+    <p>Dear {{ receiver.first_name }},</p>
+    {% if paper.state.name == 'accepted' %}
+        <p>Congratulations! A judge has <strong>accepted</strong> your paper <strong>{{ contribution.title }}</strong>.</p>
```

Comment:
Isn't "Congratulations" a bit too enthusiastic? Also, I would get rid of "A judge .." in favor of "Your paper has been ...". It doesn't matter to the user that there are judges etc, all that's important is that it has been accepted/rejected/etc